### DISCUSSION TITLE
Tidal Login

### DISCUSSION THREAD

**Discussion by skijbal** [RandomNinjaAtk/arr-scripts#234] [General]
When I login to Tidal it says the token is good for X days.  After that time has passed, do I need to log back in or is this handled automatically?

**Comment by RandomNinjaAtk** [comment]
You will likely need to login again, but it does seem to last longer than it says... Your mileage may vary...

**Comment by Coder84619** [comment]
In my experience, which is about 4-6 weeks, the token has auto renewed without any intervention from me. I think it grabs a "refresh" token which allows them to get a new token when one expires.

**Comment by skijbal** [comment]
I can confirm that it "auto renewed" overnight.
